index.html

```diff
@@ -0,0 +1,188 @@
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">
+              brunch_dining
+            </i>
+          </div>
+          <div class="programCardTitle">
+            Welcome Drinks
+          </div>
+          <div class="programCardDescription">
+            Welcoming drinks open bar: selected fizzy Aperi - wines accompanied
+            by a wide selection of locally produced gourmet appetizers
+          </div>
+        </li>
+
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">record_voice_over</i>
+          </div>
+          <div class="programCardTitle">
+            Lecture
+          </div>
+          <div class="programCardDescription">
+            Esteemed oenologists explaining the various properties of wine, guiding
+            in the choice of the right wine glass and food to match...
+          </div>
+        </li>
+
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">dinner_dining</i>
+          </div>
+          <div class="programCardTitle">
+            Dinner
+          </div>
+          <div class="programCardDescription">
+            Seasonal delicacies produced locally: - entree' of cured meats and cheeses,
+            - first course (pasta), and - main course of choice.
+          </div>
+        </li>
+
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">music_note</i>
+          </div>
+          <div class="programCardTitle">
+            Live Music
+          </div>
+          <div class="programCardDescription">
+            Live band entertaining with soft jazz. A perfect match to make your experience
+            even more sensational.
+          </div>
+        </li>
+
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">cake</i>
+          </div>
+          <div class="programCardTitle">
+            Dessert
+          </div>
+          <div class="programCardDescription">
+            To end the evening on a sweet note: Tastes of delicious light
+            pastries along with carefully selected dessert wines.
+          </div>
+        </li>
+      </ul>
+      <button type="button" class="button contact desktop-invisible">
+        Book your spot
+      </button>
+      <a href="#" class="button program upper mobile-invisible">See the whole program</a>
+    </section>
+
+    <section class="main__speakers flex flex-ai-c">
+      <h2>Experts</h2>
+      <p class="separator"></p>
+      <ul class="speakersContent flex">
+      </ul>
+      <button type="button" class="button expand upper flex flex-jc-c flex-ai-c">
+        More
+        <i class="material-icons">expand_more</i>
+      </button>
+    </section>
+  </main>
+
+  <footer class="footer mobile-invisible">
```

Comment:
- [ ] The footer of the home page doesn't appear on a mobile screen. Please fix that as asked by the previous reviewer, and kindly keep in mind styling it on the mobile screen as mentioned earlier, with a white background color 😄 🙏 
_I know that the footer doesn't appear on the image of the original design. However, it's just a presentational image and it's still expected to see a footer on the home page._ 😄